rpm/distgen.spec.dg

```diff
@@ -36,6 +36,8 @@ file and preexisting distribution metadata generate output file.
 
 %install
 %{pybin} setup.py install --root=%{buildroot}
+mkdir -p %{buildroot}%{_datadir}
+mv %{pylib}/distgen/{distconf,templates} %{buildroot}%{_datadir}
```

Comment:
Why 'install' doesn't install those files?   It might raise chances that the RPM installed code won't work, but dunno.  Probably not testable in Copr at this point unfortunately.

Reply:
Because they're not `data_files` anymore, so they install inside the `pylib` together with the package code itself and I move them to make sure the packaging is "proper" - data files moved to data directory.